clinica/utils/io.py

```diff
@@ -151,6 +151,9 @@ def check_input_caps_files(list_caps_files, caps_type, pipeline_name, caps_direc
     from colorama import Fore
 
     subject_ids = [participant_ids[i] + '_' + session_ids[i] for i in range(len(participant_ids))]
+    # 'Bug': when using PyBIDS or PyCAPS, files extracted are in reversed order
+    # compared to the given subjects and sessions
+    subject_ids = list(reversed(subject_ids))
```

Comment:
Reversed order relative to self.subjects and self.sessions ?

Reply:
Yes.